api/src/test/postman/okta_preauth.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * This file is a backup of the OSMT Postman API Collection's pre-request script.
+ */
+
+// Wrapper to chain async requests in Postman pre-request sandbox
+const _dummy = setInterval(() => {}, 300000);
+function sendRequest(req) {
+    return new Promise((resolve, reject) => {
+        pm.sendRequest(req, (err, res) => {
+            return resolve(res);
+        });
+    });
+};
+
+(async function main() {
+    // Per Okta endpoints documented at https://developer.okta.com/docs/reference/api/oidc
+
+    // Get Okta Session Token for API access
+    let authSessionUrl = pm.environment.get("oktaUrl") + "/api/v1/authn";
+    console.log(`Get Okta Session Token...`);
+    let result = await sendRequest({
+        url: authSessionUrl,
+        method: 'POST',
+        header: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json'
+        },
+        body: {
+            mode: 'raw',
+            raw: JSON.stringify({
+                username: `${pm.environment.get("oktaUsername")}`,
+                password: `${pm.environment.get("oktaPassword")}`,
+                options: {
+                    multiOptionalFactorEnroll: false,
+                    warnBeforePasswordExpired: true
+                }
+            })
+        }
+    });
+    console.log(result.json());
```

Comment:
is this an extra console.log or is it missing context message like lines 53 & 54 or 67 & 68?